pkg/controller/nodenetworkstateconfiguration/nodenetworkstateconfiguration_controller.go

```diff
@@ -200,21 +139,9 @@ func (r *ReconcileNodeNetworkStateConfiguration) Reconcile(request reconcile.Req
 
 	nmstateOutput, err := nmstate.ApplyDesiredState(instance)
 	if err != nil {
-		errmsg := fmt.Errorf("error reconciling nodenetworkstate at desired state apply: %s, %v", nmstateOutput, err)
```

Comment:
Don't remove it from here, let's have it at both controllers, it's useful for debugging purposes and to really wait for rollback to finish.